client/my-sites/subscribers/queries/use-subscribers-query.tsx

```diff
@@ -63,6 +66,10 @@ const useSubscribersQuery = ( {
 				...( sortOrder && { sort_order: sortOrder } ),
 			} );
 
+			filters.forEach( ( filter ) => {
```

Comment:
I'm appending the filters to the query in this format: `filters[]=email_subscriber&filters[]=paid`